resources/guidelines/testing/e2e-best-practises.md

```diff
@@ -1,52 +1,53 @@
 # Best practises on writing end-to-end tests
 
-A typical E2E test can be complex, with a large number of steps that take a lot of time to complete manually. Because of this complexity, E2E tests can be difficult to automate and slow to execute. The following tips can help reduce the cost and pain of E2E testing and still reap the benefits.
+A typical E2E test can be complex, with many steps that take a lot of time to complete manually. Because of this complexity, E2E tests can be difficult to automate and slow to execute. The following tips can help reduce the cost and pain of E2E testing and still reap the benefits.
 
-Cypress got you covered with their best practises as well: So please also look at their [best practises](https://docs.cypress.io/guides/references/best-practices.html) to get to know their patterns.
+Cypress got you covered with their best practises as well: So please also look at their best practises to get to know their patterns:
 
-## When should I consider writing an E2E test?
+{% embed url="https://docs.cypress.io/guides/references/best-practices.html" %}
 
-Due to running times it is not advisable to cover every single workflow available. That's why we need a way to prioritize our test cases. The following criteria may help you with that:
+{% hint style="warning" %} We strongly recommend to follow Cypress' own best practises as well. {% endhint %}
 
-* Cover the most general and most used workflows of a feature, e.g. CRUD operations. The term "happy path" describes
+## Amount and prioritisation of end-to-end tests
 
-  those workflows quite well. 
+### Video
 
-* Use risk-analysis: Cover those workflows with E2E tests, which are most vulnerable and would cause most damage 
+At [Shopware Community Day](https://scd.shopware.com/en/) 2020, we gave a talk on how we approach automated testing in Shopware, how far we have come on this journey, and what we have gained so far:
 
-  if broken. 
+{% embed url="https://www.youtube.com/watch?v=sxvQoWF4KS0" %}
 
-* Avoid duplicate coverage: E2E tests should only cover what they can cover, usually big-picture user 
+To sum it up briefly for your End-to-End testing context, End-Tto-End tests are the one tests being slow and thus expensive to maintain. That's why we need a way to prioritize our test cases.
 
-  stories \(workflows\) that contain many components and views. 
+### When should I write an end-to-end test?
 
-* Sometimes unit tests are suited better: For example, use an E2E test to test your application's reaction on a 
-
-  failed validation, not the validation itself.
+You see, due to running times it is not advisable to cover every single workflow available. The following criteria may help you with that:
+* Cover the most general, most used workflows of a feature, e.g. CRUD operations. The term "happy path" describes those workflows quite well.
```

Comment:
Maybe link to Wiki or maybe there is some better source for `happy path` ? 

The term was new to me